app/controllers/concerns/delete_enabled.rb

```diff
@@ -0,0 +1,14 @@
+# DeleteEnabled redirects the user back if delete support is not enabled. A
+# `before_action` will be created for the :destroy method.
+module DeleteEnabled
```

Comment:
There is some convention to name concerns like polymorph activerecord relations ending with `able`, e.g. `Deletable`